conbench/runner.py

```diff
@@ -159,6 +159,15 @@ def benchmark(self, f, name, publish=True, **kwargs):
 
         try:
             data, output = self._get_timing(f, iterations, timing_options)
+            # It's hard to read what this next function call really does. It
+            # does _not_ publish, but I think it creates a specific data
+            # structure. Should this be in the exception handler? Within
+            # self._get_timing() above we run user-given code, so that is
+            # expected to raise exceptions, and wants to be handled. But which
+            # exceptions is  self.record() expected to raise especially when
+            # _not_ doing HTTP interaction? And why do we handle those
+            # exceptions in the same way as those exceptions that are raised by
+            # user-given code?
```

Comment:
let's get back to this at some point, created https://github.com/conbench/conbench/issues/1070